src/docs/antora/modules/ROOT/pages/appendix.adoc

```diff
@@ -158,11 +158,22 @@ include::{jdbc-schema-base}/schema-hsqldb.sql[]
 [[schemas.mysql]]
 === MySQL
 
-[source, sql]
+[source,sql]
 ----
 include::{jdbc-schema-base}/schema-mysql.sql[]
 ----
 
+[NOTE]
```

Comment:
Note in the appendix..it's a bit unusual, but I couldn't find a better spot. Thoughts?"